src/main.py

```diff
@@ -161,15 +167,44 @@ def open_pr(subject, body, branch):
     base = "master"
     head = "{}:{}".format(repo.owner.login, branch)
     pr_message = ((body or "") + "\n\n" + DISCLAIMER).strip()
-    # Include closed PRs – if the maintainer has closed our last PR, we don't want to
+
+    if os.path.isfile("flathub.json"):
```

Comment:
Now thinking about it, I wonder if it should kept configurable at all. I can't think of a good reason why someone would want to disable auto-merge for invalid URLs.